Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>This one was tricky, well done !!!</p><p> </p><p>Would be better to calculate a column WinningTeam the winner of each game</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>